core/src/main/java/io/confluent/rest/logging/AbstractNCSARequestLog.java

```diff
@@ -156,7 +156,7 @@ public void log(Request request, Response response)
       buf.append("] \"");
       append(buf,request.getMethod());
       buf.append(' ');
-      append(buf, request.getUri().toString());
+      append(buf, request.getRequestURI());
```

Comment:
I was a bit unsure which of the methods `getOriginalURI` and `getRequestURI` is the one that replaced `getUri`.

Reply:
IIRC, I checked the debugger to see what held the correct value, but it could use some more testing if the unit tests don't cover this.

Reply:
Actually, we can get rid of this entire file. This is still around from way back in v1.0 where JDK6 requirement meant we couldn't use a new enough version of Jetty that had support for this. We missed removing it in previous upgrades. I'll just get rid of it and replace with Jetty version.